workloads/kube-burner/workloads/node-pod-density/node-pod-density.yml

```diff
@@ -81,5 +81,6 @@ jobs:
       - objectTemplate: pod.yml
         replicas: 1
         inputVars:
-          containerImage: gcr.io/google_containers/pause:3.1
+          containerImage: ${NODE_POD_DENSITY_IMAGE}
           nodeSelector: "${POD_NODE_SELECTOR}"
+          tolerations: "${POD_TOLERATIONS}"
```

Comment:
What is used this variable for? It doesn't have any default value

Reply:
It is the toleration for the `os=Windows:NoSchedule` taint added to the windows nodes and is default empty to it is not added to other workloads.
It is set from the [DAG](https://github.com/cloud-bulldozer/airflow-kubernetes/pull/250/files#diff-1ed8b3795e32ef8e399751c11b8b44dc549a696cf9640045dd025fd8e43bc7efR20).

Alternatively: Since it is only a toleration, we could leave it in place permanently without a variable.

Reply:
Yeah, I like your seccond approach, let's leave w/o a variable it as it doesn't affect the default behavior in linux nodes

Reply:
Sounds good. Removed it.
FWIW: the mcr image manifest appears to have linux and windows images, so we can do tests on both windows and linux workers in the same run.